examples/with-loading/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "with-loading",
+  "version": "0.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "../../dist/bin/next"
```

Comment:
This requires a top-level `npm install`, which is not clarified in the instructions. I think it's best if we include maybe `"next": "*"` in the deps?